patches/@mantine+core+7.16.3.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs b/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs
+index d0006e5..49e2259 100644
+--- a/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs
```

Comment:
i patched both the cjs and mjs file, so that we're covered in case we change something in how we build metabase or the sdk